scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -182,6 +182,10 @@
           SPIDEX
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
+        <li class="list-group-item">
```

Comment:
Perhaps we could have a condition here, like {% if variant.spliceai_ds_max %} so the li is not shown if there's no value in the field? But perhaps it's informative knowing that there is no spliceai. In that case you could write later in the span:
<span class="float-right">{{ variant.spliceai_ds_max or "-"}} </span>

Reply:
Sure! I'll do the same for spidex!

Reply:
It turns out spidex has a solution where the value is always looked up, and spidex_human set to "not_reported" when not found. To each his own I guess.. Chalk one for refactor. I currently like your solution better so I'll use it.